pkg/config/sink.go

```diff
@@ -221,14 +221,14 @@ func (c *CSVConfig) validateAndAdjust() error {
 	case 0:
 		return cerror.WrapError(cerror.ErrSinkInvalidConfig,
 			errors.New("csv config delimiter cannot be empty"))
-	case 1, 2:
+	case 1, 2, 3:
```

Comment:
Perhaps we should add a delimiter size limit, which only allows a maximum of two 2-character delimiter by default, but users can turn on this configuration to use more delimiter

Reply:
I guess it's not a good action to add a new configuration for this small behavior. Because it will increase unnecessary mental burden on the user.

Reply:
I don't suggest adding another config at this moment. Maybe we can consider adding a new config like "max-delimiter-length" if there is really necessary in the future.